lib/core/lib/generators/avro/avro_generator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative './avro_builder.rb'
+
+class AvroGenerator < Rails::Generators::Base
+  def create_files
+    Ros.table_names.each do |name|
```

Comment:
the avro generator does not operate on all models/tables in a service. It operates on one model, like all the other generators

Reply:
This needs to work this way because it should be being invoked by the endpoint generator which is scoped to the single endpoint